### DISCUSSION TITLE
1 Tahun Sebagai Product Engineer di Dicoding Indonesia - Rahmat Agung Julians

### DISCUSSION THREAD

**Discussion by giscus** [rahmatagungj/public-rahmatagungjulians.tech#14] [Comments]
# 1 Tahun Sebagai Product Engineer di Dicoding Indonesia - Rahmat Agung Julians

Perjalanan mencari pengalaman dengan bekerja sebagai Product Engineer di EdTech Startup

https://www.rahmatagungjulians.tech/blog/1-tahun-sebagai-product-engineer-di-dicoding-indonesia#alasan-bergabung

**Comment by bagussatoto** [comment]
Semangat bang agung 
 ----
Pencapaian anda luar biasa,
semoga mendapatkan pekerjaan yang lebih baik lagi,
Sukses terus

**Comment by ghost** [comment]
Sekedar saran kak, untuk code editornya boleh coba typeface seperti Cascadia atau Fira code supaya lebih menarik.

**Reply by rahmatagungj** [reply]
Untuk font code interaktif ya?

**Reply by ghost** [reply]
yups
